DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -78,6 +81,20 @@ function PreFirstEntryMapFixes()
         break;
     case "02_NYC_WAREHOUSE":
         if (VanillaMaps){
+            foreach RadiusActors(class'CrateUnbreakableSmall', crateSmall, 0.0, vectm(-1658.93, 664.61, -358.68)) {
+                crateSmall.bIsSecretGoal = true;
+                loc = crateSmall.Location;
+                loc.y -= 130.0;
+                crateSmall.SetLocation(loc);
+                break; // does this actually save any cycles with a zero radius?
```

Comment:
idk but using a 0 radius might not be safe depending how the physics work? especially in other mods where our code runs 1 frame later I think, better to use something like 8 radius (half a foot)